flex/validation/parameter.py

```diff
@@ -51,6 +51,8 @@ def type_cast_parameters(parameter_values, parameter_definitions, context):
             parameter_definition = find_parameter(parameter_definitions, name=key)
         except KeyError:
             continue
+        except ValueError:
+            continue
```

Comment:
Yes - it's needed if the request contains a parameter that's not specified in the swagger schema. I couldn't find anything about passing undeclared parameters (something like `additionalParameters`?), but I believe it's legal in Swagger v2.0. 

So, rather than failing over with `ValueError` or converting it into a `ValidationError`, the validation should just ignore such fields.